uint8 tank_id
float64 x
float64 y

float64 vx
float64 vy

float64 basis_yaw
float64 bat_yaw

float64 basis_yawdot
float64 bat_yawdot
